src/rules/gameshell/gameshell.tsx

```diff
@@ -29,24 +31,24 @@ export const Shell: GameRuleInterface = {
                 const header = React.createElement(Shell.views['HostShell-Main-Head'], this.props);
                 const body = React.createElement(Shell.views['HostShell-Main-Body'], this.props);
 
-                return React.DOM.div({id: 'shell-main'},
-                                     header,
-                                     body);
+                return React.DOM.div({ id: 'shell-main' },
+                    header,
+                    body);
             }
         },
 
         'HostShell-Main-Head': class extends React.Component<ViewPropsInterface, {}> {
             public render() {
-                return React.DOM.div({id: 'shell-header'},
-                                     React.DOM.div({id: 'shell-room'},
-                                                   this.props.MP.roomId));
+                return React.DOM.div({ id: 'shell-header' },
```

Comment:
fix indentation (this is exactly why arguments should be aligned with the indentation of previous argument)